releases/release-1.29/release-notes/maps/pr-119385-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 119385
+releasenote:
+  text: Introduced new apiserver metric `apiserver_flowcontrol_current_inqueue_seats`.
+    This metric is analogous to `apiserver_flowcontrol_current_inqueue_requests`,
+    but tracks totals seats as each request can take more than 1 seat.
```

Comment:
Good point. Having read the PR and looking at the code, it seems to be about providing the number of "seats" (the analogy being around a train), but since each request can actually take up more than one seat, the total number of seats will be higher than the number of requests. I agree that the language can be improved, I'll go with "total number of seats".